interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: In a class, super inputs an argument that is organic or inherited from the parent class to the child class. 
 
-Researched answer:
+Researched answer: In a class, the super method is utilized in the subclass to use instance variables from the superclass, or the parent. Instance variables cannot be inherited from the parent because instance variables are "local" to a speicfic object. After calling on super, the child will then have permissions to the instance variables nested inside that method.
 
 2. What is a gem?
 
-Your answer:
+Your answer: Gems are installed for Ruby. Similar to node modules in Jest for testing JavaScript. 
 
-Researched answer:
+Researched answer: A gem in Ruby is a conglomoration of code snippet functionality. The Ruby package manger, RubyGems, manages gems, similar to the yarn package manager for JavaScript testing. Gems allows programmers to add features without creating the same code over and over again on an application. Gems ensures functionality and easy dissemination between programmers. 
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A relational database is a one-stop-shop that holds attributes and data that correlate in at least one way or another to each other. Relational databases are very useful when trying to return a very specific amount or type of data required, instead of looking at big data without any goal or task. Relational databases help with organization and flow and remove the clusstered mess of unassociated and unneccessary data from an environment. 
 
-Researched answer:
+Researched answer: A relational database depicts information in rows and columns, similar to an Excel spreadsheet. This type of database holds organized data in a predefined relationship, which allows for easier visibility on the relationship between different types of data. SQL is a common type of relational databases. Objects written in Ruby almost mirror data stored in the columns and rows in relational databases. Ruby's objects conists of key value pairs, which can be reflected in relational databases. PostgreSQL is also another type of relational database, as it is an object-oreinted database. PostgreSQL is very similar to sQL, just an advanced version, with more functionality and relations between objects.
```

Comment:
Make sure to address what other kinds of databases there are that are not relational.